doc/coding_standards.md

```diff
@@ -390,6 +390,16 @@ extern "C" {
 #include <limits>
 ```
 
+### Singletons
+
+Do not introduce new singleton classes. Prefer to move code in the direction
+of fewer singleton classes over time.
+
+Singletons complicate destruction, complicate making multiple instances of
+something in the future, i.e. having two Tox profiles loaded at once is
```

Comment:
"e.g."

Reply:
I merged too fast :(

You're right, but I guess it'll stay for now.